### PR TITLE
fixes #501 Kill the offline popup, and replace it by an inline card

### LINKED ISSUE #501: Kill the offline popup, and replace it by an inline card
image

Kill the offline popup, and replace it by an inline card that can be dismissed (and that facts is remembered and honoured using settings)

### PR DESCRIPTION
**Pull request by Shubham2320**
## Description

Added a CardView (which can be dismissed) to show the respective message.
And the dismissal message is stored in sharedPreference for future use.
Also, the message will not appear if an item is present in the history list.

## Related issues and discussion
fixes #501  

 ## Screen-shots, if any

### CODE REVIEW

File: app/src/main/res/layout/fragment_offline_edit.xml [hunk 1]
```diff
@@ -1,14 +1,77 @@
...
+        android:padding="@dimen/padding_large"
+        app:layout_constraintBottom_toTopOf="@+id/buttonSendAll"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:background="#fff"
```
Comment (policorn) [inline app/src/main/res/layout/fragment_offline_edit.xml]: `@color/white` here please

File: app/src/main/res/layout/fragment_offline_edit.xml [hunk 2]
```diff
@@ -1,14 +1,77 @@
...
+            android:layout_height="wrap_content">
+
+            <ImageButton
+                android:id="@+id/message_dismiss_icon"
+                android:layout_width="@dimen/button_height_too_short"
+                android:layout_height="@dimen/button_height_too_short"
+                android:layout_alignParentRight="true"
+                android:background="#fff"
```
Comment (policorn) [inline app/src/main/res/layout/fragment_offline_edit.xml]: `@color/white` here please

File: app/src/main/res/layout/fragment_offline_edit.xml [hunk 3]
```diff
@@ -1,14 +1,77 @@
...
+                android:orientation="vertical">
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:padding="@dimen/padding_short"
+                    android:text="@string/title_info_dialog"
+                    android:textAlignment="center"
+                    android:textSize="18sp"
```
Comment (policorn) [inline app/src/main/res/layout/fragment_offline_edit.xml]: `@dimen/font_large`

File: app/src/main/res/layout/fragment_offline_edit.xml [hunk 4]
```diff
@@ -1,14 +1,77 @@
...
+                    android:textStyle="bold" />
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:textSize="14sp"
```
Comment (policorn) [inline app/src/main/res/layout/fragment_offline_edit.xml]: Can it be font_normal? (1sp difference)

File: app/src/main/res/layout/fragment_offline_edit.xml [hunk 5]
```diff
@@ -26,9 +89,17 @@
...
         android:background="@drawable/rounded_button"
-        android:text="@string/txtSendAll"
-        android:paddingStart="50dp"
         android:paddingEnd="50dp"
-        android:textSize="16sp" />
-</RelativeLayout>
\ No newline at end of file
+        android:paddingStart="50dp"
+        android:text="@string/txtSendAll"
+        android:textSize="16sp"
```
Comment (policorn) [inline app/src/main/res/layout/fragment_offline_edit.xml]: font_normal?

**Comment by Shubham2320** [comment]
@policorn I've made the changes please review :).

**Comment by policorn** [comment]
Approved. Thanks @Shubham2320! Merging...

**Pull request merged**